### PR TITLE
Update Wired-7 support

### PR DESCRIPTION
**Pull request by Wired-7**
Due to our imageboard configuration, trying to get thread with .html extension returns a 404 error. This error snowballs into users not being able to post from Kuroba.
And I take the opportunity to remove an unused board.

### CODE REVIEW

**Review by K1rakishou — APPROVED** [review]

**Comment by Wired-7** [comment]
Please do not merge the pull request yet. I'm changing a few thing on the server and it may not be necessary.

File: Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/sites/Wired7.java [hunk 1]
```diff
@@ -82,7 +82,6 @@ public void setup() {
         setIcon(SiteIcon.fromFavicon(HttpUrl.parse("https://wired-7.org/favicon.ico")));
 
         setBoards(
-                Board.fromSiteNameCode(this, "Lewds & +18", "18"),
```
Comment (Adamantcheese) [inline Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/sites/Wired7.java:85]: If you're deleting a board, you'll need to update DatabaseHelper and use the deleteBoard method. See https://github.com/Adamantcheese/Kuroba/commit/ebdae105581cf727ef1a679ee33c3b87fae5a4c9.

**Comment by K1rakishou** [comment]
@Wired-7 
Hi. I noticed that wired-7 sometimes sends incorrect file md5 hash. Could you please take a look at it? You don't have to do it in this PR.

Here is a thread - https://wired-7.org/a/res/601.html#q601
These two files have incorrect md5 hash that  is being send in thread.json
https://wired-7.org/a/src/1571092676114.jpg
https://wired-7.org/a/src/1571131472923.jpg

Just take a file md5 of those files and compare it to what is being sent in thread.json:
The first file has hash "c42c40fad9a27033eb2df426952ca8af" in thread json, but if you download the file and calculate the hash by yourself you will get the following hash: "D87EA5B037BE1342FBEC2A474F53526D" which is completely different.

**Comment by Wired-7** [comment]
> 
> 
> @Wired-7
> Hi. I noticed that wired-7 sometimes sends incorrect file md5 hash. Could you please take a look at it? You don't have to do it in this PR.
> 
> Here is a thread - https://wired-7.org/a/res/601.html#q601
> These two files have incorrect md5 hash that is being send in thread.json
> https://wired-7.org/a/src/1571092676114.jpg
> https://wired-7.org/a/src/1571131472923.jpg
> 
> Just take a file md5 of those files and compare it to what is being sent in thread.json:
> The first file has hash "c42c40fad9a27033eb2df426952ca8af" in thread json, but if you download the file and calculate the hash by yourself you will get the following hash: "D87EA5B037BE1342FBEC2A474F53526D" which is completely different.

What you are referring as hash of first file, actually is the filename. The json has the correct info:
https://wired-7.org/a/res/601.json
That image is from the first post with the _"filename"_ key as _"c42c40fad9a27033eb2df426952ca8af"_.
Maybe is Kuroba changing something?
I didn't change the api so it's weird that only happens in Wired-7, it should happen on all Vichan sites.

File: Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/sites/Wired7.java (hunk 1 above)
Comment (Wired-7) [reply Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/sites/Wired7.java:85]: Got it, thanks.

**Comment by Adamantcheese** [comment]
K1rak messed up mentioning the filename by accident, but his point still stands. The OP image has the wrong md5 being sent over, which should be (in hex) what K1rak put down, or `2H6lsDe+E0L77CpHT1NSbQ==` in base64. The md5 implementation is outside of our control, as it's in the Okio library, but it is correct, comparing what K1rak put against the linux utility md5sum.

**Comment by Adamantcheese** [comment]
Looking at some Vichan source code, I would ask you to check if `$config['bsd_md5']` or `$config['gnu_md5']` are set, because otherwise it uses the PHP md5 implementation, which may be what's causing the issue? There's also this function here https://github.com/vichan-devel/vichan/blob/0aa4e3badc346aba27edbaa71da5966642ab060a/inc/api.php#L90, which seems to allow for some sort of different hash to be used? I'm not entirely sure if those are helpful though.

**Comment by K1rakishou** [comment]
Yeah, I've mixed them up. I meant this: "md5":"xCxA+tmicDPrLfQmlSyorw==". And it's incorrect for just these two posts (Out of like 10 I'd checked).

>it should happen on all Vichan sites.

Well we have rolled out FileCacheV2, which uses file hash comparison, couple of days ago and there weren't any complains (yet). And I tested it myself for more than a week and haven't seen it on 4chan (neither on sushchan/lainchan). Out of all sites we support (that use Vichan) I only noticed it on wired-7.

Apparently filename is the same same the md5 hash (in hex) which is "c42c40fad9a27033eb2df426952ca8af". That's why I've mixed them up.

**Pull request merged**